cddl/cmw-example-tag-1668576819-def.cddl

```diff
@@ -0,0 +1,5 @@
+$cbor-tag /= cbor-tag<1668576819, my-evidence>
```

Comment:
In what cddl file should the `$cbor-tag` extension that is specific to my favorite evidence type be located? Should the spec the defines `my-evidence` be updated to include the definition or should it be added to some other cddl file?